Storage/container/one_to_one.h

```diff
@@ -207,47 +222,47 @@ class GenericOne2One {
   Iterator End() const { return Iterator(map_.end()); }
 
  private:
-  void DoErase(const T_RCPAIR& row_col) {
-    forward_.erase(row_col.first);
-    transposed_.erase(row_col.second);
-    map_.erase(row_col);
+  void DoErase(const T_KEY& key) {
+    forward_.erase(key.first);
+    transposed_.erase(key.second);
+    map_.erase(key);
   }
 
-  void DoAdd(const T_RCPAIR& row_col, const T& object) {
-    auto& placeholder = map_[row_col];
+  void DoAdd(const T_KEY& key, const T& object) {
+    auto& placeholder = map_[key];
     placeholder = std::make_unique<T>(object);
-    forward_[row_col.first] = placeholder.get();
-    transposed_[row_col.second] = placeholder.get();
+    forward_[key.first] = placeholder.get();
+    transposed_[key.second] = placeholder.get();
   }
 
-  T_WHOLE_MAP map_;
+  T_ELEMENTS_MAP map_;
   T_FORWARD_MAP forward_;
   T_TRANSPOSED_MAP transposed_;
   MutationJournal& journal_;
 };
 
 template <typename T, typename T_UPDATE_EVENT, typename T_DELETE_EVENT>
-using UnorderedOne2One = GenericOne2One<T, T_UPDATE_EVENT, T_DELETE_EVENT, Unordered, Unordered>;
+using UnorderedOneToOne = GenericOneToOne<T, T_UPDATE_EVENT, T_DELETE_EVENT, Unordered, Unordered>;
 
 template <typename T, typename T_UPDATE_EVENT, typename T_DELETE_EVENT>
-using OrderedOne2One = GenericOne2One<T, T_UPDATE_EVENT, T_DELETE_EVENT, Ordered, Ordered>;
+using OrderedOneToOne = GenericOneToOne<T, T_UPDATE_EVENT, T_DELETE_EVENT, Ordered, Ordered>;
```

Comment:
Done, see [this commit](https://github.com/C5T/Current/pull/448/commits/463d9e6b73378855917e11d78e0175fb818cd1c8). May be, it's not enough, but still better than nothing